website/snippets/_privatelink-hostname-restriction.md

```diff
@@ -0,0 +1,5 @@
+:::caution Environment variables
+
+Using [Environment variables](/docs/build/environment-variables) when configuring PrivateLink endpoints isn't supported in dbt Cloud. Instead, use [Extended Attributes](/docs/deploy/deploy-environments#extended-attributes) to dynamically change these values in your dbt Cloud environment.
```

Comment:
This is only _sorta true,_ since it _technically_ works in ST.  There's some nuance I guess though in that in ST, you dont configure to use a PL endpoint in-app, where as you do in MT.  Do we foresee this causing extra confusion/issues for customers due to this difference?  

To be clear though, I'm more than happy for us to just say across the board, regardless of ST vs MT, that we just dont want folks to use env vars for their connection hostnames if they use PL.

Reply:
https://dbt-labs.slack.com/archives/C04RSP33V8U/p1704382058007459?thread_ts=1704377784.263199&cid=C04RSP33V8U